src/reducers/dataSets/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+    LOADING_DATA_SETS_DONE,
+    LOADING_DATA_SETS_ERROR,
+    LOADING_DATA_SETS_START,
+    SET_DATA_SETS_FILTER,
+} from './actions'
+
+export * from './actions'
+export * from './selectors'
+export * from './thunks'
+
+export const dataSetsDefaultState = {
+    error: '',
+    data: [],
+    loaded: false,
+    loading: false,
+    filter: '',
+}
+
+export default function dataSets(
+    state = dataSetsDefaultState,
+    { type, payload } = {}
```

Comment:
The reducer will never be called without an action right? Or can that happen?

Reply:
Removed the confusing default value